src/app/docs/page-summary/page-summary-docs.component.html

```diff
@@ -0,0 +1,22 @@
+<sky-docs-demo-page
+  moduleName="SkyPageSummarytModule"
+  moduleSourceCodePath="src/app/public/modules/page-summary/"
+  pageTitle="Page summary"
+>
+  <sky-docs-demo-page-summary>
+    The page summary component displays critical information and actions for users to access quickly and frequently.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="UX guidelines"
+    >
```

Comment:
I'd recommend wrapping the content in a `sky-docs-design-guideline` element like we do elsewhere, and I'd add a `heading="Toolbar"` attribute to it.